scene/3d/cpu_particles_3d.cpp

```diff
@@ -1097,7 +1097,7 @@ void CPUParticles3D::_update_particle_data_buffer() {
 			ptr[10] = t.basis.elements[2][2];
 			ptr[11] = t.origin.z;
 		} else {
-			memset(ptr, 0, sizeof(float) * 12);
+			memset(ptr, 0, sizeof(real_t) * 12);
```

Comment:
Does this correspond to the size of `Transform3D`? Can we maybe use `sizeof(Transform3D)` directly to be safe?